### PR TITLE
Test for project service Java side

### PR DESCRIPTION
**Pull request by jose2a**
I’ve changed Project Service: -> Everything marked with Test Team 2019

Src:
com.revature.services.ProjectService

Tests:
com.revature.controllers.CreateProjecIntegrationTest
com.revature.services.ProjectServiceTestSuite
*****************
Tested Project submission, and made some needed changes that were validated in client side but not in server side.
Implemented and tested -> Newly submitted projects should have a status of "Pending" and should not be displayed on the  "All Projects" view until they have been approved by an admin user.

### CODE REVIEW

File: src/main/java/com/revature/security/CustomAuthenticationFilter.java [hunk 1]
```diff
@@ -124,10 +124,11 @@ public String get_SHA_512_SecureHash(String passwordToHash, String salt) {
 	 * @param header The retrieved header from the request object
 	 */
 	public boolean validateHeader(String header) {
-		if (header == null) {
-			return false;
-		}
-		return header.equals(get_SHA_512_SecureHash(zuulConfig.getSecret(), zuulConfig.getSalt()));
+//		if (header == null) {
```
Comment (jurczakn) [inline src/main/java/com/revature/security/CustomAuthenticationFilter.java]: Same issue, add code back in

**Review by jurczakn — CHANGES_REQUESTED** [review]
Make the change and should be good

File: src/main/java/com/revature/security/CustomAuthenticationFilter.java (hunk 1 above)
Comment (jose2a) [reply src/main/java/com/revature/security/CustomAuthenticationFilter.java]: Fixed

**Review by jurczakn — APPROVED** [review]
Good, work around removed

**Pull request merged**